### ISSUE TITLE
Multiple issues with xdebug on php 8.0.x & 8.1.x - #1 - segfaults

### ISSUE THREAD

**Issue by sbulen** [easyphp/DEVSERVER#255]
Note that the file for xdebug 3.1.0 faults, so debugging does not work for the php versions that use it (8.0.11, 8.0.12, 8.0.13).  

```
Faulting application name: eds-httpserver.exe, version: 2.4.43.0, time stamp: 0x5ea00a5d
Faulting module name: php_xdebug-3.1.0-8.0-vs16-x86_64.dll, version: 3.1.0.1, time stamp: 0x615acc41
Exception code: 0xc0000005
Fault offset: 0x000000000000c840
Faulting process id: 0x2e48
Faulting application start time: 0x01d7e3ba116575be

```

The 3.0.4 version of xdebug in prior php versions (8.0.9) works fine.  

This issue has been reported elsewhere.   (The problem is with that particular xdebug binary, not EasyPHP, but it means that EasyPHP should not use the 3.1.0 binaries going forward...)

The workaround if you need to use the step debugger is to go back to 8.0.9.  OR, download the 3.1.1 version from the xdebug site & edit eds-app-settings.php accordingly.

**Comment by sbulen** [comment]
Note that the outward facing symptom you see here if you attempt to enable the debugger in xdebug manager usually appears to be a cert error...  E.g.:
`An error occurred during a connection to localhost. PR_CONNECT_RESET_ERROR`

But this is a total red herring...  The real issue is that the 3.1 module ID'd above fails and is not loading.

**Comment by sbulen** [comment]
I see a similar fault on 8.1.  This is in the event viewer:
```
Faulting application name: eds-httpserver.exe, version: 2.4.43.0, time stamp: 0x5ea00a5d
Faulting module name: php_xdebug-3.1.0-8.1-vs16-x86_64.dll, version: 3.1.0.1, time stamp: 0x615acc3d
Exception code: 0xc0000005
Fault offset: 0x000000000000cb50
Faulting process id: 0x7d0
Faulting application start time: 0x01d7e475d2b6f98f

```

**Comment by sbulen** [comment]
Note that the latest available (3.1.1) on the xdebug download site works as well: php_xdebug-3.1.1-8.1-vs16-x86_64.dll

**Comment by easyphp** [comment]
We will switch to 3.1.1. Thanks!
